src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -54,6 +57,8 @@ const BG_COLORS_BY_DARK_MODE_AND_CHAIN_ID: {
       'radial-gradient(100% 93.36% at 0% 6.64%, rgba(160, 108, 247, 0.1) 0%, rgba(82, 32, 166, 0.1) 100%)',
     [SupportedChainId.POLYGON_MUMBAI]:
       'radial-gradient(100% 93.36% at 0% 6.64%, rgba(160, 108, 247, 0.1) 0%, rgba(82, 32, 166, 0.1) 100%)',
+    [SupportedChainId.CELO]: 'rgba(120, 250, 160, 0.55)',
+    [SupportedChainId.CELO_ALFAJORES]: 'rgba(120, 250, 160, 0.55)',
```

Comment:
@ppphhhiiilll any thoughts on a Celo gradient? This would just leave the UI a solid color, which is a departure from the current pattern.